mp/src/game/shared/basecombatweapon_shared.h

```diff
@@ -314,6 +314,9 @@ class CBaseCombatWeapon : public BASECOMBATWEAPON_DERIVED_FROM
 	virtual void			WeaponSound( WeaponSound_t sound_type, float soundtime = 0.0f );
 	virtual void			StopWeaponSound( WeaponSound_t sound_type );
 	virtual const WeaponProficiencyInfo_t *GetProficiencyValues();
+#ifdef NEO
+	virtual float			GetPenetration();
+#endif
```

Comment:
We could declare this in `CNEOBaseCombatWeapon` so we don't need to ifdef the Valve base class for it.